microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -93,31 +111,39 @@ If you need to access more detailed information, use the [getHighEntropyValues()
    :::column-end:::
 :::row-end:::  
 
-## Recommended uses of User-Agent Client Hints  
-
-The set of brands and the associated display order change over time, so you should never hard-code indices into the array of returned brands.  To help ensure you spot similar issues in your website early, the browser includes a `GREASE` brand value that changes over time and is formatted to break because of common string parsing issues.  
+For more information navigate to [getHighEntropyValues()][GithubWicgUaClientHintsGethighentropyvalues].
 
-If you're prevented from using [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection], don't use a hardcoded list of known Chromium-based browsers for verification.  Examples of hardcoded browser names include `Microsoft Edge` and `Google Chrome`.  Reasons why [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] isn't applicable may include the following situation.  
+### Using feature detection with User Agent Client Hints
 
-*   A fix for a Chromium bug in a later version must be avoided and the affected browsers are difficult to detect outside of a verification of the brand and version.  
-    
-Instead, you should verify the `Chromium` brand, which ensures your detection applies to all affected Chromium-based browsers.  
+Combining User-Agent Client Hints with [feature detection][MdnLearnToolsTestingCrossBrowserTestingFeatureDetection] is the best mechanism for browser identification. Microsoft recommends using it whenever possible to:

```

Comment:
I mentioned earlier in the review that feature detection is not quite the same thing as browser detection. Using both things is generally good web design, but I don't know if we need to call that out explicitly. Should we omit this section?

Reply:
I have no idea ... maybe you and @jenny-microsoft and @Reezaali can reach a consensus on this one ...

Reply:
Alright, how's this:

- We change the title of this section to something like "User-Agent Client Hints Best Practices" or "User-Agent Client Hints Tips"
- We rephrase this particular part to read:

> Combining User-Agent Client Hints with feature detection is an effective way to deliver compatible web content. Microsoft recommends using this pattern to:

- After the bulleted list, we add a sentence before "Use this method to verify.." that reads something like this (to clarify the code example):

> In addition, Microsoft recommends detecting Chromium, the engine that powers both Microsoft Edge and Google Chrome, when it is necessary to check for a Chrome-like browser.

Reply:
+1. Let's keep references to feature detection to a minimum. The general point we want to make is "if you can use feature detection, don't read this article and don't use user agent for browser detection".